go.mod

```diff
@@ -3,21 +3,21 @@ module github.com/integr8ly/cloud-resource-operator
 go 1.17
 
 require (
-	github.com/aws/aws-sdk-go v1.39.2
+	github.com/aws/aws-sdk-go v1.44.24
```

Comment:
we might need to bump it further as per https://github.com/integr8ly/cloud-resource-operator/pull/418